common/flatpak-dir.c

```diff
@@ -8407,15 +8407,18 @@ flatpak_dir_user_check_parental_controls (FlatpakDir    *self,
    * anyone. */
   if (app != NULL)
     {
-      g_autoptr(EpcAppFilter) app_filter = NULL;
-      app_filter = epc_get_app_filter (dbus_connection, getuid (), TRUE, cancellable, error);
+      g_autoptr(MctManager) manager = NULL;
+      g_autoptr(MctAppFilter) app_filter = NULL;
+
+      manager = mct_manager_new (dbus_connection);
+      app_filter = mct_manager_get_app_filter (manager, getuid (), TRUE, cancellable, error);
```

Comment:
Please use  a `MctAppFilterFlags` here now that we have in malcontent. 
```suggestion
      app_filter = mct_manager_get_app_filter (manager, getuid (), MCT_GET_APP_FILTER_FLAGS_INTERACTIVE, cancellable, error);
```
Also can you put the suggestion in the upstream PR, otherwise it will fall though the cracks.